Gruntfile.js

```diff
@@ -33,66 +47,57 @@ module.exports = function(grunt) {
         ]
-      },
-      tracker: {
-        src: 'src/spid-tracker.js'
-      }
-    },
-    concat: {
-      options: {
-        stripBanners: true,
-        banner: '<%= banner %>',
-        process: true
-      },
-      sdk: {
-        src: ['<%= jshint.sdk.src %>'],
-        dest: 'dist/spid-sdk-<%= pkg.version %>.js'
-      },
-      sdktracker: {
-        src: ['<%= jshint.sdk.src %>', '<%= jshint.tracker.src %>'],
-        dest: 'dist/spid-sdk-pulse-<%= pkg.version %>.js'
       }
     },
     uglify: {
       options: {
         banner: '<%= banner %>'
       },
-      sdk: {
-        src: '<%= concat.sdk.dest %>',
-        dest: 'dist/spid-sdk-<%= pkg.version %>.min.js'
+      sdkVar: {
+        src: 'dist/<%= pkg.version %>/spid-sdk-<%= pkg.version %>-var.js',
+        dest: 'dist/<%= pkg.version %>/spid-sdk-<%= pkg.version %>-var.min.js'
+      },
+      sdkAmd: {
+        src: 'dist/<%= pkg.version %>/spid-sdk-<%= pkg.version %>-amd.js',
+        dest: 'dist/<%= pkg.version %>/spid-sdk-<%= pkg.version %>-amd.min.js'
       },
-      sdktracker: {
-        src: '<%= concat.sdktracker.dest %>',
-        dest: 'dist/spid-sdk-pulse-<%= pkg.version %>.min.js'
+      sdkCommonJs: {
+        src: 'dist/<%= pkg.version %>/spid-sdk-<%= pkg.version %>-commonjs2.js',
+        dest: 'dist/<%= pkg.version %>/spid-sdk-<%= pkg.version %>-commonjs2.min.js'
       }
     },
-    blanket_mocha: {
-      all: [ 'test/index.html' ],
-      options: {
-        threshold: 70
+
+    karma: {
+      unit: {
+        configFile: 'karma.conf.js'
       }
     },
-    watch: {
-      gruntfile: {
-        files: '<%= jshint.gruntfile.src %>',
-        tasks: ['jshint:gruntfile']
-      },
-      sdk: {
-        files: ['<%= jshint.sdk.src %>', '<%= jshint.tests.src %>'],
-        tasks: ['jshint:sdk', 'jshint:tracker', 'jshint:tests', 'blanket_mocha']
-      }
+
+    /*
+    webpack : {
+      var: webpackCfg('var', 'SPiD'),
+      amd: webpackCfg('amd', 'SPiD'),
+      commonjs: webpackCfg('commonjs2', 'SPiD')
+    }
+    */
+    webpack : {
+      varSdk: webpackCfg('var', 'spid-sdk', 'SPiD'),
+      varUri: webpackCfg('var', 'spid-uri', 'SPiD_Uri'),
+      amdSdk: webpackCfg('amd', 'spid-sdk', 'SPiD'),
+      amdUri: webpackCfg('amd', 'spid-uri', 'SPiD_Uri'),
+      commonJsSdk: webpackCfg('commonjs2', 'spid-sdk', 'SPiD'),
+      commonJsUri: webpackCfg('commonjs2', 'spid-uri', 'SPiD_Uri')
     }
   });
 
   // These plugins provide necessary tasks.
+  grunt.loadNpmTasks("grunt-webpack");
   grunt.loadNpmTasks('grunt-contrib-uglify');
-  grunt.loadNpmTasks('grunt-contrib-concat');
   grunt.loadNpmTasks('grunt-contrib-jshint');
-  grunt.loadNpmTasks('grunt-contrib-watch');
-  grunt.loadNpmTasks('grunt-blanket-mocha');
+  grunt.loadNpmTasks('grunt-karma');
 
   // Default task.
-  grunt.registerTask('default', ['jshint:sdk', 'jshint:tracker', 'concat', 'uglify']);
+  grunt.registerTask('default', ['jshint:sdk', 'webpack', 'uglify']);
```

Comment:
I'm considering adding the "grunt-release-it" as the way to make releases. But we could also just go with the NPM variant of release it.